test/test_core.py

```diff
@@ -7222,13 +7222,20 @@ def test_dyncall_specific(self, *args):
       print(str(args) + ' ' + which)
       self.do_core_test('dyncall_specific.c', emcc_args=['-D' + which] + list(args) + extra_args)
 
+  @also_with_wasm_bigint
   def test_getValue_setValue(self):
     # these used to be exported, but no longer are by default
     def test(out_suffix='', args=None, assert_returncode=0):
```

Comment:
How about renaming `out_suffix=''` to `asserts=False` and then the setting of the suffic would all be in this inner function?

The reason I suggest this is because Its not clear to me how this works when `test('_assert'` is called in wasmbigint versions of the test?